boards/frdm-k64f/Makefile.dep

```diff
@@ -0,0 +1,4 @@
+ifneq (,$(filter ng_netif,$(USEMODULE)))
+  USEMODULE += kw2xrf
```

Comment:
If I see it right, the board does natively not have a radio soldered on, right? So you should remove this file.